app/Models/User.php

```diff
@@ -1531,6 +1531,15 @@ public function scopeOnline($query)
             ->whereRaw('user_lastvisit > UNIX_TIMESTAMP(DATE_SUB(NOW(), INTERVAL '.config('osu.user.online_window').' MINUTE))');
     }
 
+    public function scopeEagerloadForListing($query)
+    {
+        return $query->with([
+            'country',
+            'supporterTags',
```

Comment:
thanks to recent change, this should be `supporterTagPurchases` :dancer:

Reply:
/tableflip